src/NotificationEventMailing.php

```diff
@@ -283,10 +283,7 @@ public static function send(array $data)
                                 trigger_error('Failed to add document ' . $doc->fields['filepath'] . ' to mail: file not found', E_USER_WARNING);
                                 continue;
                             }
-                            $image_path = Document::getImage(
-                                GLPI_DOC_DIR . "/" . $doc->fields['filepath'],
-                                'mail'
-                            );
+                            $image_path = Document::getImage(GLPI_DOC_DIR . "/" . $doc->fields['filepath']);
```

Comment:
If there is no need to resize, then there is no need to call any method.
```suggestion
                            $image_path = GLPI_DOC_DIR . "/" . $doc->fields['filepath'];
```

Reply:
done !